src/core/src/pass/manager.cpp

```diff
@@ -52,7 +52,7 @@ void ov::pass::Manager::run_passes(shared_ptr<ov::Model> func) {
     NGRAPH_SUPPRESS_DEPRECATED_START
     OV_ITT_SCOPED_TASK(ov::itt::domains::nGraph, "pass::Manager::run_passes");
 
-    static bool profile_enabled = ov::util::getenv_bool("NGRAPH_PROFILE_PASS_ENABLE");
+    static bool profile_enabled = ov::util::getenv_bool("OV_PROFILE_PASS_ENABLE");
```

Comment:
should we keep old name for compatibility?

Reply:
I will support both names